modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/side_panel/SidePanel.js

```diff
@@ -329,11 +329,12 @@ export default class SidePanel extends React.Component {
 	}
 
 	render() {
-		const visibility = this.state.visible ? 'is-visible' : 'is-hidden';
 		const loading =
 			this.state.loading || (this.state.moving && this.state.visible)
 				? 'is-loading'
 				: '';
+		const moving = this.state.moving ? 'is-moving' : '';
```

Comment:
`this.state` - 🤯 been a few years since I've seen a class component

Reply:
Ya looking at this file I forgot about doing things like `this.handleIframeSubmit = this.handleIframeSubmit.bind(this);` 🙃

Reply:
![](https://media1.giphy.com/media/84BjZMVEX3aRG/200.gif)

Reply:
[not blocking] This could be inline to `classNames`, something like
```js
className={classNames(
		'is-moving': this.state.moving,
)}
```

Reply:
I wanted to group these `is-*` css classes together so that's why I didn't make this inline.